### DISCUSSION TITLE
Mobx snapshot/serializer : Is serializr dead?

### DISCUSSION THREAD

**Discussion by jtwigg** [mobxjs/mobx#3450] [Q&A] [answered]
**What are people using to (de)serialize their state? Possibly to localstore or DB?**

1) I'm trying to use `serializr` but the project is stale by 2 years and not working well under any conditions on the latest React Scripts + Webpack. **Is the project dead?**
2) I can use mobx-keystone, but thats a heavy lift for my needs, and pulling in another sparsely maintained project (700*'s github)

**Comment by mweststrate** [comment, marked as answer]
serializr is basically a finished project. I don't see any direct reason why it shouldn't work with webpack anymore in general. It might not work with react-scripts, since they have a very specific babel config, that might not work together with the in the readme documented configuration, if you are trying to use the decorator syntax.
